Packages/com.unity.inputsystem/LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Input System © 2024 Unity Technologies
+com.unity.inputsystem copyright © 2024 Unity Technologies
```

Comment:
Is this a recommended update ?

Both seem ok so no major issue with this.

Reply:
looked at what the other packages were doing and most had this naming. the word "copyright" was the warning trigger for the validation suite